app/src/main/java/com/group15/fitnesstracker/onboarding/login/LoginPresenter.kt

```diff
@@ -0,0 +1,34 @@
+package com.group15.fitnesstracker.onboarding.login
+
+import android.annotation.SuppressLint
+import android.content.Context
+import com.group15.fitnesstracker.R
+import com.group15.fitnesstracker.base.BaseView
+import com.group15.fitnesstracker.db.dao.UserDao
+import com.group15.fitnesstracker.util.Constants
+import io.reactivex.schedulers.Schedulers
+
+class LoginPresenter(val view: LoginContract.View, private val context: Context, private val userDao: UserDao): LoginContract.Presenter {
+    override fun start() {
+        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
```

Comment:
Necessary?

Reply:
yes, for now. `BasePresenter` has that method. Not really sure why i added it, but can be removed